provider/aws.go

```diff
@@ -271,12 +269,39 @@ func changesByZone(zones map[string]*route53.HostedZone, changeSet []*route53.Ch
 	return changes
 }
 
+func expandAliasTargets(eps []*endpoint.Endpoint) (endpoints, aliasTargets []*endpoint.Endpoint) {
+	for _, ep := range eps {
+		targets := []string{}
+		for _, target := range ep.Targets {
+			if ok := isAWSLoadBalancer(ep.RecordType, target); ok {
```

Comment:
nit: you could simplify this to just `if isAWSLoadBalancer(ep.RecordType, target) {`